manifests/init.pp

```diff
@@ -201,7 +202,7 @@
   Optional[String] $package_source                                 = undef,
   Optional[String] $package_provider                               = undef,
   $refclocks                                                       = [],
-  $peers                                                           = [],
+  Variant[Hash,Array[Stdlib::Host]] $peers                         = [],
```

Comment:
Should it be something like this:
```suggestion
  Variant[Hash[Stdlib::Host, Optional[String]],Array[Stdlib::Host]] $peers                         = [],
```

Reply:
I made it the same as the type for `server` and `pool`.  I think it'd actually something like...
`Variant[Hash[Stdlib::Host, Optional[Array[String]]],Array[Stdlib::Host]]`

But perhaps greater readability in the REFERENCE.MD is more important than having a super strict type??

Reply:
PR updated with extra commit that adds new type alias.